test/test_connection.py

```diff
@@ -153,6 +150,14 @@ def test_ping_pong(self, as_client):
             data = wire_data[2:]
         assert data == payload
 
+        # Verify that connection vends the PongReceive event with the correct payload
```

Comment:
I think the argument for using it goes that "sends" implies the actual I/O process, which this library does not do, so "vends" (as in, what a vending machine does) is an appropriate alternative.

To native english speakers I think it sounds weird: "to vend" is a verb not usually used outside of the context of the noun phrase "vending machine" or the noun "vendor", so it clangs into the mind. For my part I think that's good enough reason not to use it: if it will confuse native English speakers then we'll get a bunch of PRs that attempt to change it.

However, I do agree that the general idea of wanting to avoid the word "send" is reasonable. In this case "emit" might work better.